packages/amplify-e2e-tests/src/__tests__/rds-v2-generate-schema.test.ts

```diff
@@ -74,9 +74,10 @@ describe('RDS Generate Schema tests', () => {
       region,
     };
     const queries = [
-      'CREATE TABLE Contact (ID INT PRIMARY KEY, FirstName VARCHAR(20), LastName VARCHAR(50))',
-      'CREATE TABLE Person (ID INT PRIMARY KEY, Info JSON NOT NULL)',
-      'CREATE TABLE tbl_todos (ID INT PRIMARY KEY, description VARCHAR(20))',
+      'CREATE TABLE Contact (id INT PRIMARY KEY, firstName VARCHAR(20), lastName VARCHAR(50))',
```

Comment:
Why are we changing the field names from PascalCase to camelCase?

Reply:
Like I mentioned in the above comment, some of the existing tests have table setups that will now be generated with a `refersTo` if they are not camel cased which increases the scope of what they're testing. So I added additional tests for `refersTo` on fields cases and kept the existing ones simpler.